From Maurice@suse.de  Thu Feb 25 15:33:57 2010
From: Maurice Dawson <maurice2699@btinternet.com>
Date: Fri, 19 Feb 2010 10:53:13 +0000
Subject: Staging: comedi: fix brace coding style issue in ni_labpc.c
Message-ID: <1266576795-3109-1-git-send-email-maurice@ubuntu.ubuntu-domain>


This is a patch to the ni_labpc.c file that fixes up a brace warning
found by the checkpatch.pl tool.

Signed-off-by: Maurice Dawson <maurice2699@btinternet.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/comedi/drivers/ni_labpc.c
+++ b/drivers/staging/comedi/drivers/ni_labpc.c
@@ -483,12 +483,10 @@ int labpc_common_attach(struct comedi_device *dev, unsigned long iobase,
 
 	printk("comedi%d: ni_labpc: %s, io 0x%lx", dev->minor, thisboard->name,
 	       iobase);
-	if (irq) {
+	if (irq)
 		printk(", irq %u", irq);
-	}
-	if (dma_chan) {
+	if (dma_chan)
 		printk(", dma %u", dma_chan);
-	}
 	printk("\n");
 
 	if (iobase == 0) {
-- 
1.6.3.3

